services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/ApplicationEntity.kt

```diff
@@ -73,4 +74,10 @@ data class ApplicationEntity(
             hankeId = hanke.id,
         )
     }
+
+    fun allContacts(): List<HankekayttajaEntity> =
```

Comment:
One would assume a method with this name would return contacts (like HakemusyhteyshenkiloEntity).